FreeBSD/usr/local/share/ceph/example/ceph-build-cluster.sh

```diff
@@ -0,0 +1,654 @@
+#!/bin/sh
```

Comment:
@tchaikov 
We could do that instead.
Putting it in examples is a bit more "free from support" under FreeBSD, because it is just what it is: an example. 
But you'd still then want to make it `src/script/FreeBSD`, as not to confuse other OSes?
I'm using ZFS, Jails and some other FreeBSD tricks, so it'll not work on Linux.